.ci/validation.sh

```diff
@@ -569,26 +569,32 @@ jdk14-assembly-site)
   mvn -e --no-transfer-progress site -Pno-validations
   ;;
 
-# executed only in wercker for security reasons
 sonarqube)
   # token could be generated at https://sonarcloud.io/account/security/
   # execution on local for master:
-  # SONAR_TOKEN=xxxxxx ./.ci/wercker.sh sonarqube
+  # SONAR_TOKEN=xxxxxx ./.ci/validation.sh sonarqube
   # execution on local for non-master:
-  # SONAR_TOKEN=xxxxxx PR=xxxxxx WERCKER_GIT_BRANCH=xxxxxx ./.ci/wercker.sh sonarqube
-  if [[ $PR && $PR =~ ^([0-9]*)$ ]]; then
-      SONAR_PR_VARIABLES="-Dsonar.pullrequest.key=$PR"
-      SONAR_PR_VARIABLES+=" -Dsonar.pullrequest.branch=$WERCKER_GIT_BRANCH"
+  # SONAR_TOKEN=xxxxxx PR_NUMBER=xxxxxx PR_BRANCH_NAME=xxxxxx ./.ci/validation.sh sonarqube
```

Comment:
If these 3 are required for the script to function, then we need to add something to verify they are applied.
Example: https://github.com/checkstyle/checkstyle/blob/master/.ci/update-github-page.sh#L14

Reply:
Only SONAR_TOKEN is required, I have moved this check above to be first step in script

Reply:
So the script will function 100% with `SONAR_PR_VARIABLES` not set? It only gets set currently if `PR_NUMBER` is set.

Reply:
Yes, these variables are only used for PRs, sonar execution on master build does not need this.

Reply:
Ok, then resolving.